src/Simala/Expr/Parser.hs

```diff
@@ -39,7 +39,7 @@ keyword k =
 
 identifier :: Parser Text
 identifier =
-  Text.cons <$> satisfy isAlpha <*> takeWhileP (Just "identifier char") (\ x -> isAlphaNum x || x `elem` ("_" :: String))
+  Text.cons <$> satisfy (\c -> isAlpha c || c == '#') <*> takeWhileP (Just "identifier char") (\ x -> isAlphaNum x || x `elem` ("_" :: String))
```

Comment:
My intuition is that `#eval` should *not* be a normal identifier. I see that `keyword` is using `identifier` though, but perhaps `#eval` should not be `keyword` either, but `directive`, with `directive` essentially being `char '#'` followed by `keyword`.